12LessonTwelve/in-class-project/index.html

```diff
@@ -1,10 +1,45 @@
 <!DOCTYPE html>
-<html>
-<head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
-</head>
-<body>
+ <html>
+ <head>
+     <title>lesson 12</title>
+     <meta charset="utf-8">
+     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+     <meta name="viewport" content="width=device-width, initial-scale=1">
+     <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
+     <link rel="stylesheet" href="css/style.css">
+ </head>
+ <body>
+     <header class="row text-center">
+         <div id="order">
+             <h1>Order Takeaway or Delivery Food</h1>
+             <p>So good its really good forget about it</p>
+         </div>
+     </header>
+     <section class="row">
+         <div class="col-sm-6 col-xs-12 col-lg-3 search">
+           <img src="img/icon_home_2.svg" />
+             <h2>Search by address</h2>
+             <p>Find all restaurants available in your zone</p>
+         </div>
+         <div class="col-sm-6 col-xs-12 col-lg-3 search">
+           <img src="img/icon_home_1.svg" />
+             <h2>Search by address</h2>
+             <p>Search by gps</p>
+         </div>
+         <div class="col-sm-6 col-xs-12 col-lg-3 search">
+           <img src="img/icon_home_3.svg" />
+             <h2>Search by address</h2>
+             <p>Search in your area</p>
+         </div>
+         <div class="col-sm-6 col-xs-12 col-lg-3 search">
+           <img src="img/icon_home_4.svg" />
+             <h2>Search by address</h2>
+             <p>Search by the area</p>
+         </div>
+     </section>
+     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
+
 
 </body>

```

Comment:
Way too much space between icons in the phone view. Make this change and I can bump your grade up.
<img width="375" alt="screen shot 2016-09-13 at 5 57 05 pm" src="https://cloud.githubusercontent.com/assets/12698279/18494430/ae89a5d0-79db-11e6-8b7b-5c341727592c.png">